src/ui/components/shared/PortalDropdown.tsx

```diff
@@ -49,6 +49,7 @@ export default function PortalDropdown(props: PortalDropdownProps) {
         className={`expand-dropdown flex w-full ${buttonStyle}`}
         disabled={props.disabled}
         id="portal-dropdown-button"
+        data-test-id="consoleDockButton"
```

Comment:
Why is this test id on a component called `PortalDropdown` that's used in a few places (like Workspace members, etc)

Mistake?